src/commands/deploy/notifyDeployComplete.ts

```diff
@@ -46,19 +46,19 @@ async function listHttpTriggerUrls(context: IActionContext, node: SlotTreeItemBa
         const functions: AzExtTreeItem[] = await functionsNode.getCachedChildren(context);
         const anonFunctions: RemoteFunctionTreeItem[] = <RemoteFunctionTreeItem[]>functions.filter((f: AzureTreeItem) => f instanceof RemoteFunctionTreeItem && f.config.isHttpTrigger && f.config.authLevel === HttpAuthLevel.anonymous);
         if (anonFunctions.length > 0) {
-            ext.outputChannel.appendLine(localize('anonymousFunctionUrls', 'HTTP Trigger Urls:'));
+            ext.outputChannel.appendLog(localize('anonymousFunctionUrls', 'HTTP Trigger Urls:'));
             for (const func of anonFunctions) {
-                ext.outputChannel.appendLine(`  ${func.label}: ${func.triggerUrl}`);
+                ext.outputChannel.appendLog(`  ${func.label}: ${func.triggerUrl}`);
```

Comment:
I would use `appendLine` as it's listing out all the URLs. The timestamp is already displayed for the header above